zerver/webhooks/front/view.py

```diff
@@ -39,27 +39,27 @@ def get_target_name(payload: WildValue) -> str:
 
 
 def get_inbound_message_body(payload: WildValue) -> str:
-    link, outbox, inbox, subject = get_message_data(payload)
+    link, outbox, inbox, topic = get_message_data(payload)
     return (
         "[Inbound message]({link}) from **{outbox}** to **{inbox}**:\n"
-        "```quote\n*Subject*: {subject}\n```".format(
-            link=link, outbox=outbox, inbox=inbox, subject=subject
+        "```quote\n*Subject*: {topic}\n```".format(
```

Comment:
Not changing `Subject` in the returning string, since it breaks the test `test_inbound_message`.

Reply:
Yes, you are right. As the word `Subject` is used in a message and not in a variable name. Hence, there's nothing to be done here.